client/src/components/game/phases/events/EventCard.vue

```diff
@@ -80,6 +94,28 @@ export default class EventCard extends Vue {
     return this.active && this.gamePhase === this.phase.events;
   }
 
+  private eventNoChangeViews: Array<EventClientView> = [
+    'NO_CHANGE',
+    'AUDIT',
+    'DISABLE_CHAT'
+  ];
+
+  get requiresInteraction() {
+    if(!this.eventNoChangeViews.includes(this.event.clientViewHandler)){
```

Comment:
this could be simplified to 

`return !this.eventNoChangeViews.includes(this.event.clientViewHandler)`